app/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanDao.java

```diff
@@ -59,6 +60,11 @@ final void setCriteriaDao(CriteriaDao criteriaDao) {
         DynamoDBQueryExpression<DynamoSchedulePlan> query = new DynamoDBQueryExpression<DynamoSchedulePlan>();
         query.withScanIndexForward(false);
         query.withHashKeyValues(plan);
+        if (!includeDeleted) {
+            query.withQueryFilterEntry("deleted", new Condition()
+                .withComparisonOperator(ComparisonOperator.NE)
+                .withAttributeValueList(new AttributeValue().withN("1")));
+        }
```

Comment:
Unit test for includeDeleted vs not includeDeleted?

Reply:
Added to the mock version of the DAO tests.